test/CMakeLists.txt

```diff
@@ -8,6 +8,13 @@ message("Building tests")
 
 # libdnf
 add_subdirectory(libdnf)
+
+# performance tests are available only for libdnf
+if(WITH_PERFORMANCE_TESTS)
+    return()
+endif()
```

Comment:
So `WITH_PERFORMANCE_TESTS` enables building performance tests in addition to the regular test, doesn't it? So why turn off building all the other tests when it's on? It shouldn't be turning off anything?

Reply:
If you do performance testing, you don't want to get your output spammed with results of regular unit tests. That's why it's either-or.

Reply:
My CMake knowledge limitations come into play here, but seems to me you should make it a different target then, if don't actually want them to run together ever? Seems easier to use and less confusing.